packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -125,6 +125,16 @@ export interface PipelineProps {
    * @default true
    */
   readonly crossAccountKeys?: boolean;
+
+  /**
+   * Enables KMS key rotation for cross-account keys
+   *
+   * By default kms key rotation is disabled, but will add an additional $1/month
+   * for each year the key exists when enabled.
+   *
+   * @default false
+   */
+  readonly crossAccountKeyRotationEnabled?: boolean;
```

Comment:
I think the name of this property is misleading. It seems to imply it only applies to to the cross-account Keys, but it actually applies to all Keys (including the single-account Key generated when a Bucket was not provided).

Let's rename this to remove the "cross-account" part. Maybe something simple like `enableKeyRotation?: boolean`?

Reply:
Ok cool, will update the property name accordingly. Missed that it was for all keys